core/appmodule/v2/module.go

```diff
@@ -104,3 +104,14 @@ type ValidatorUpdate struct {
 type HasRegisterInterfaces interface {
 	RegisterInterfaces(registry.InterfaceRegistrar)
 }
+
+// ModuleAccount defines a module account info get from the module.
+type ModuleAccount struct {
```

Comment:
It seems this is only would be for genesis module accounts. when a chain creates them dynamically this would fail because it may not be present correct? Maybe we name is something to identify genesis accounts or put it in the go doc

Reply:
wdym create them dynamically, like we create new module accounts besides the genesis one ?